chiselFrontend/src/main/scala/chisel3/core/Aggregate.scala

```diff
@@ -10,11 +10,12 @@ import chisel3.internal._
 import chisel3.internal.Builder.pushCommand
 import chisel3.internal.firrtl._
 import chisel3.internal.sourceinfo.{SourceInfo, DeprecatedSourceInfo, VecTransform, SourceInfoTransform}
+import chisel3.NotStrict.CompileOptions
```

Comment:
Do we want non strict compile for base Chisel classes? (at least that's how I'm interpreting scoping rules for strict vs. non-strict)